widgets/tab/tabdemo/tabdemo.go

```diff
@@ -0,0 +1,548 @@
+// Package main provides a system monitor application using tabbed interfaces.
```

Comment:
This widget has a really good potential. Playing a bit with the demo, I think we have some instability that we should track down if you feel like diving in and debugging.

The instability is visible if I am just clicking through the tabs (or just repeatedly pressing tab). Occasionally the tab switches to the previoustab instead to the next and occasionally it switches over multiple tabs. This is likely some racy behavior (lack of locking / serializing of the events).

I did not look at the code in detail. Let me know if you have the cycles to debug the behavior.

Reply:
Actually looking deeper, this is just caused by a goroutine in the demo that periodically switches tabs to a random tab.

How would you feel about removing that goroutine as it makes the demo feel unstable. No weird behavior is observed once that goroutine is removed.

Reply:
It's happening because FollowNotifications has been set to true in the demo, notifications generated by the app will cause a warning symbol to appear on the tab itself notifying the user that some activity or alarm has triggered. FollowNotifications(True) just follows those series of notifications and switches focus/tabs for you. I can certainly set it to False so people don't get confused. Let me know if that makes sense.